api/auth/auth_handler.py

```diff
@@ -1,17 +1,21 @@
 import time
 import jwt
+from jwt import PyJWKClient
 from decouple import config
 
 
 JWT_SECRET = config("JWT_SECRET")
 JWT_ALGORITHM = config("JWT_ALGORITHM")
 JWT_AUDIENCE = config("JWT_AUDIENCE")
+OIDC_CERTS_URL = config("OIDC_CERTS_URL")
 
 
 def token_response(token: str):
     return {"access_token": token}
 
 
 def decode_token(token: str) -> dict:
-    decoded_token = jwt.decode(token, JWT_SECRET, audience=JWT_AUDIENCE, algorithms=[JWT_ALGORITHM])
+    jwks_client = PyJWKClient(OIDC_CERTS_URL)
+    signing_key = jwks_client.get_signing_key_from_jwt(token)
+    decoded_token = jwt.decode(token, signing_key.key, audience=JWT_AUDIENCE, algorithms=[JWT_ALGORITHM])
```

Comment:
Does it make sense to replace JWT_AUDIENCE for OIDC_CLIENT_ID here as well since they are the same value? And then get rid of JWT_AUDIENCE

Reply:
They aren't always the same, apparently. In the decoded access_token from Keycloak, for example, the `aud` is actually 'account' and then the client-id is elsewhere in the json object. So, I think we may want to leave it for now.

Reply:
Oh I see, got it, thanks!